app/views/admin/dashboard/index.html.erb

```diff
@@ -1 +1,10 @@
 <h1> Hello, <%= current_user.name %>! Welcome to your Admin Dashboard!</h1>
+
+<% @orders.order_by_status.each do |order| %>
+<section id = "order-<%= order.id %>">
+  <%= link_to order.user.name, "/admin/users/#{order.user.id}" %><br>
+  <%= order.id %><br>
+  <%= order.created_at %><br>
+  <%= order.status %>
```

Comment:
According to the story, I don't think `order.status` needs to display?

Reply:
Appears so! I just assumed because they wanted it sorted that way that they'd want it displayed, but I think you're right.

Reply:
I think it's fine if it's displayed for now?